app/routes/events/components/EventDetail/index.tsx

```diff
@@ -326,18 +327,21 @@ export default class EventDetail extends Component<Props, State> {
       event.responsibleGroup && {
         key: 'Arrangør',
         value: (
-          <span>
-            {groupLink ? (
-              <Link to={groupLink}>{event.responsibleGroup.name}</Link>
-            ) : (
-              event.responsibleGroup.name
+          <Link to={groupLink} className={styles.organizerLogo}>
+            {groupLink && (
+              <Tooltip
+                key={event.responsibleGroup.id}
+                content={event.responsibleGroup.name}
+              >
+                <CircularPicture
+                  alt={event.responsibleGroup.name}
+                  src={event.responsibleGroup.logo}
+                  size={40}
+                />
+              </Tooltip>
             )}{' '}
-            {event.responsibleGroup.contactEmail && (
-              <a href={`mailto:${event.responsibleGroup.contactEmail}`}>
-                {event.responsibleGroup.contactEmail}
-              </a>
-            )}
-          </span>
+            {event.responsibleGroup.name}
+          </Link>
```

Comment:
hmm is the tooltip really needed now that the name is next to the image? I suggest either removing it or putting the email address inside.

Reply:
I shall remove it. The email looks very weird inside it 👍